### DISCUSSION TITLE
Joining Tables that Don't Have a Foreign Key Relationship

### DISCUSSION THREAD

**Discussion by Geobot19** [PostgREST/postgrest#3154] [Q&A] [answered]
### Environment

* Supabase 

### Description of issue
I was attempting to replicate this command
`SELECT profiles.name, stories.title, stories.description, stories.created_on FROM stories INNER JOIN profiles on profiles.id = stories.user`
using supabase-js when I realized that there is no way to do an inner join on two tables that don't have a foreign key relationship
(`profiles.id` and and `stories.user` both reference auth.users.id, but the supabase api doesn't let you look at auth tables that way)

I think that there should be a way to join any two tables regardless of their relationship. 

I was told to ask the PostgREST team for this feature by people on the supabase discord

**Comment by steve-chavez** [comment, marked as answer]
Try https://postgrest.org/en/stable/references/api/resource_embedding.html#computed-relationships
